build.sbt

```diff
@@ -77,57 +26,20 @@ def npmModulesTarballTask = Def.task {
   Seq(zipFile)
 }
 
-lazy val itServer = sys.env.get("PLAY_VERSION") match {
-  case Some("2.8") => itServerPlay28
-  case Some("2.9") => itServerPlay29
-  case _           => itServerPlay30
-}
-
-lazy val itServerPlay30 = Project("it-server-play-30", file("it-server-play-30"))
+lazy val itServer = (project in file("it-server-play-30"))
   .enablePlugins(PlayScala, SbtWeb)
   .dependsOn(playNunjucksPlay30)
-  .settings(inConfig(Test)(testSettings): _*)
-  .settings(sharedItServerSettings: _*)
-  .settings(scalaVersion := scala2_13)
-  .settings(libraryDependencies ++= LibDependencies.itServerPlay30)
-
-lazy val itServerPlay29 = Project("it-server-play-29", file("it-server-play-29"))
-  .enablePlugins(PlayScala, SbtWeb)
-  .dependsOn(playNunjucksPlay29)
-  .settings(copySources(itServerPlay30))
-  .settings(inConfig(Test)(testSettings): _*)
-  .settings(sharedItServerSettings: _*)
-  .settings(scalaVersion := scala2_13)
-  .settings(libraryDependencies ++= LibDependencies.itServerPlay29)
-
-lazy val itServerPlay28 = Project("it-server-play-28", file("it-server-play-28"))
-  .enablePlugins(PlayScala, SbtWeb)
-  .dependsOn(playNunjucksPlay28)
-  .settings(copySources(itServerPlay30))
-  .settings(inConfig(Test)(testSettings): _*)
-  .settings(sharedItServerSettings: _*)
   .settings(
-    scalaVersion := scala2_12,
-    crossScalaVersions := Seq(scala2_12, scala2_13)
-  )
-  .settings(libraryDependencies ++= LibDependencies.itServerPlay28)
-
-lazy val sharedItServerSettings = Seq(
-  Concat.groups := Seq(
-    "javascripts/application.js" -> group(Seq("lib/govuk-frontend/govuk/all.js"))
-  ),
-  Assets / pipelineStages := Seq(concat, uglify),
-  coverageEnabled := false
-)
-
-lazy val testSettings: Seq[Def.Setting[_]] = Seq(
-  fork := true,
-  javaOptions ++= Seq(
-    "-Dconfig.resource=test.application.conf"
+    publish / skip := true,
```

Comment:
moved this into here, since we don't want to publish integration test sub project